Crashlytics/Protogen/nanopb/crashlytics.nanopb.h

```diff
@@ -15,7 +15,7 @@
  */
 
 /* Automatically generated nanopb header */
-/* Generated by nanopb-0.3.9.7 */
+/* Generated by nanopb-0.3.9.8 */
```

Comment:
There is no difference in generation, but technically we should be on 0.3.9.9 now.

Reply:
Lemme follow up with this